rasa/core/actions/action.py

```diff
@@ -1206,6 +1208,29 @@ async def run(
         return validated_events
 
 
+def _check_mapping_validity(mapping: Dict[Text, Any], domain: "Domain") -> bool:
```

Comment:
I think this code should rather sit in `SlotMapping` class, either as a separate public method, or private method used by the `validate` method in case we even want to raise `InvalidDomain` for this type of issue?